packages/typespec-python/src/emitter.ts

```diff
@@ -104,6 +113,6 @@ export async function $onEmit(context: EmitContext<PythonEmitterOptions>) {
     }
     commandArgs.push("--from-typespec=true");
     if (!program.compilerOptions.noEmit && !program.hasError()) {
-        execFileSync(process.execPath, commandArgs);
+        exec(commandArgs.toString());
```

Comment:
```suggestion
        execSync(commandArgs.join(" "));
```

Reply:
1. `exec` is async, so we're not actually waiting for it to call. This is why it didn't catch the issue with the venv path above
2. We can't use `toString()`, that joins the commands with ",". Instead, we want to join them with spaces